OpenKh.Command.TexFooter/Subcommands/ExportCommand.cs

```diff
@@ -0,0 +1,107 @@
+using McMaster.Extensions.CommandLineUtils;
+using OpenKh.Command.TexFooter.Models;
+using OpenKh.Command.TexFooter.TypeConverters;
+using OpenKh.Command.TexFooter.Utils;
+using OpenKh.Common;
+using OpenKh.Kh2;
+using OpenKh.Kh2.TextureFooter;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Linq;
+using System.Text;
+using YamlDotNet.Serialization;
+
+namespace OpenKh.Command.TexFooter.Subcommands
+{
+    [HelpOption]
+    [Command(Description = "map file: export map or mdlx texture footer. map -> yml")]
+    public class ExportCommand
+    {
+        [Required]
+        [FileExists]
+        [Argument(0, Description = "Map file")]
+        public string MapFile { get; set; }
+
+        [Argument(1, Description = "Output dir")]
+        public string OutputDir { get; set; }
+
+        protected int OnExecute(CommandLineApplication app) => OnExecute();
```

Comment:
Unfortunately having a second `OnExecute` makes the application crashing

Reply:
Thanks, fixed!